service-front/module/Application/view/layout/twig/layout.twig

```diff
@@ -152,7 +152,7 @@
     </script>
 
     <script src="/assets/v2/js/libs/jquery.min.js"></script>
-    <script src="{{ StaticAssetPath( '/assets/v2/js/application.js', { 'minify':true } ) }}"></script>
```

Comment:
Should minify be re-enabled now? (assuming you switched it off for debugging purposes)

Reply:
Yer, I've reverted this back.